.github/workflows/ci.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.8, 3.9]
```

Comment:
The tests crashed when using python version 3.10. This is not related to the code I have implemented, therefore I only removed this python version. However, I think this should be solved, but maybe that's part of another PR?

Reply:
I wanted to see what the crash was about with 3.10 so pushed a change where I added it back. It looks like the tests pass anyway, I think previous fails were just because the other jobs failed :)

Reply:
okay, but when I run the tests locally I get that the same one fails(?), do you experience the same?
But will have a look on this in another PR